docs/quick-start/hi-kin-python.md

```diff
@@ -47,7 +57,7 @@ print('\nEnvironment: ')
 pprint(vars(client.environment))
 ```
 ###### Output:
-
+[Don't push, your .png doesn't match with your line code. It's missing "First we will....blah blah blah." . Also the image has a "Use existing seed?". You should crop that as it is used in the next block of code]
```

Comment:
Sorry, I don't get this comment. The image shows the output of the code. What is not consistent?

Reply:
It's not like a big issue, but in each picture you have print statements that are not in the "output" screen shot when you display the code. It was confusing at first because I saw print messages in the code, but couldn't find where the print statements were being called until I scrolled up for some of them.